core/settings/qPrefDisplay.cpp

```diff
@@ -41,6 +41,9 @@ static const QByteArray st_windowState_default;
 int qPrefDisplay::st_lastState;
 static int st_lastState_default = false;
 
+bool qPrefDisplay::st_singleColumnPortrait;
+static bool st_singleColumnPortrait_default = false;
```

Comment:
You appear to be following a scheme in this file, but shouldn't it be
```static const bool st_singleColumnPortrait_default = false;```
or even
```static const constexpr bool st_singleColumnPortrait_default = false;```
?
Not a practical issue though - any modern compiler will recognize that you never change the thing and remove it.

Reply:
I really just followed the pattern. I assume that you will rip all of this out, soon.

Reply:
> I assume that you will rip all of this out, soon.

There are more important things, but it is on the todo list, yes. :-P